build.gradle

```diff
@@ -82,8 +86,21 @@ subprojects {
     }
 }
 
+detekt {
+    toolVersion = "1.22.0"
+
+    source = files("ground/src", "sharedTest/src")
+    allRules = true
+    config = files("${project.rootDir}/config/detekt/detekt.yml")
+    buildUponDefaultConfig = true
+    ignoreFailures = false
+
+    basePath = projectDir
+}
+
+// TODO: Remove java only plugins to improve build speed
```

Comment:
I think you can remove them straightaway. We shouldn't be writing new Java code, and existing code passes checks.

Reply:
We still need checkstyle for license header. But rest can be removed